client/data/reader/recommendations/use-is-in-recommend-list.ts

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from 'calypso/state';
+import { getMatchingItem } from 'calypso/state/reader/lists/selectors';
+import { useRecommendedList } from './use-recommend-list';
+
+//TODO: Replace it by react-query + move this logic to the server side
+/**
+ * Short check if a feed is in the recommended list
+ * @param owner - The owner of the recommended list
+ * @param feedId - The ID of the feed to check if is in the recommended list
+ * @returns True if the feed is in the recommended list, false otherwise
+ */
+export const useIsInRecommendedList = ( owner: string, feedId: number ) => {
```

Comment:
Does this hook we use on subscriptions work for this?
`import { useRecommendedSite } from 'calypso/landing/subscriptions/hooks/use-recommended-site';` 
`const { isRecommended, toggleRecommended } = useRecommendedSite( Number( feed_id ) );`

https://github.com/Automattic/wp-calypso/blob/98f5ac39711abbbca3bfcafc71538ac140d18a12/client/landing/subscriptions/components/site-subscriptions-list/site-subscription-row.tsx#L106